reference/api-reference/anbox-https-api.md

```diff
@@ -773,3 +774,42 @@ anbox_gralloc_buffer_allocations_total 121
 anbox_vulkan_buffer_memory_size_total 78370816
 ...
 ```
+
+(sec-anbox-https-api-telephony)=
+### `/1.0/telephony`
+#### GET
+
+* Description: Get telephony status
+ * Operation: sync
+ * Return: Current telephony status
```

Comment:
Superfluous space
```suggestion
* Operation: sync
* Return: Current telephony status
```

Reply:
This is how the other entries in the document are formatted too.